src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.csproj

```diff
@@ -6,7 +6,7 @@
     <ShouldWriteSigningRequired>false</ShouldWriteSigningRequired>
     <AllowReferenceFromRuntime>true</AllowReferenceFromRuntime>
     <RuntimeProjectFile>$(ProjectDir)\external\test-runtime\XUnit.Runtime.depproj</RuntimeProjectFile>
-    <PackageTargetFramework>netstandard2.0</PackageTargetFramework> 
+    <PackageTargetFramework>netstandard2.0;$(UAPvNextTFM)</PackageTargetFramework>
```

Comment:
This should get undone before merging to release/2.1 since they are already using the new nuget mappings. I will try to fix this before the merge.